lib/capybara/screenshot/diff/stabilization.rb

```diff
@@ -143,16 +139,18 @@ def prepare_page_for_screenshot(timeout:)
         end
 
         def take_right_size_screenshot(comparison, crop:)
+          driver = comparison.driver
+
           save_screenshot(comparison.new_file_name)
 
           # TODO(uwe): Remove when chromedriver takes right size screenshots
-          reduce_retina_image_size(comparison.new_file_name, comparison.driver)
+          reduce_retina_image_size(comparison.new_file_name, driver)
           # ODOT
 
           if crop
-            full_img = comparison.driver.from_file(comparison.new_file_name)
-            area_img = full_img.crop(crop[0], crop[1], crop[2] - crop[0], crop[3] - crop[1])
-            comparison.driver.save_image_to(area_img, comparison.new_file_name)
+            full_img = driver.from_file(comparison.new_file_name)
+            area_img = driver.crop([crop[0], crop[1], crop[2] - crop[0], crop[3] - crop[1]], full_img)
```

Comment:
@donv maybe we should not invent our own convention for this method interface, but reuse chumkypng and VIPs style: top, left, width, height? JS also returns in the same way.